lib/python/test/testsuite/Reader/_manager.py

```diff
@@ -0,0 +1,70 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2022 PIConGPU contributors
+Authors: Mika Soren Voss
+License: GPLv3+
+"""
+
+from . import dataReader
+from . import jsonReader
+from . import paramReader
+# from . import cmakeflagReader
+import testsuite._checkData as cD
+
+json = {}
+param = {}
+data = {}
+
+
+def mainsearch(dataDirection: str = None,
+               paramDirection: str = None,
+               jsonDirection: str = None):
+
+    global json, param, data
+
+    # first read jsonParameter
+    if (cD.checkExistVariables(variable="jsonDirection") or
+            jsonDirection is not None):
+
+        jsonDirection = cD.checkDirection(variable="jsonDirection",
+                                          direction=jsonDirection)
+
+        params = cD.checkVariables(variable="json_Parameter")
+        for parameter in params:
+            json[parameter] = jsonReader.getValue(parameter,
+                                                  direction=jsonDirection)
+
+    # read param Parameter
+    if (cD.checkExistVariables(variable="paramDirection") or
+            paramDirection is not None):
+
+        paramDirection = cD.checkDirection(variable="paramDirection",
+                                           direction=paramDirection)
+
+        params = cD.checkVariables(variable="param_Parameter")
+        for parameter in params:
+            param[parameter] = paramReader.getValue(parameter,
+                                                    direction=paramDirection)
+
+        # if (cD.checkExistVariables(variable="dataDirection") or
+            # dataDirection is not None):
+            # dataDirection = cD.checkDirection(variable="dataDirection",
+            # direction=dataDirection)
+            # cmakeflagReader.checkflagInDir(direction=dataDirection)
+
+    # read .dat values
```

Comment:
optional:
```suggestion
    # read data values
```
since not all input needs to be stored in a `*.dat` file (if given an explicit file name (as suggested in the next file))